perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -286,7 +286,9 @@ sub by_category_ajax_data : Private {
     # unresponsive must return empty string if okay, as that's what mobile app checks
     if ($type eq 'one' || ($type eq 'all' && $unresponsive)) {
         $body->{unresponsive} = $unresponsive;
-        if ($type eq 'all' && $unresponsive) {
+        # Check for no bodies here, because if there are any (say one
+        # unresponsive, one not), can use default display code for that.
+        if ($type eq 'all' && $unresponsive && !@$bodies) {
```

Comment:
if `$type` is `'all'` will `$unresponsive` not always be true here?

Reply:
Yes; I assume that was to just be clear it was matching the second condition of the outer if statement. But now I've added a new bit to it, not much point.